### PR TITLE
Implement dcs parameter

### PR DESCRIPTION
**Pull request by algra4**
Hello, 

Into the documentation of cm.com. We can specify a custom DCS (https://developers.cm.com/messaging/docs/sms#custom-dcs). 

However, your implementation doesn't offert this parameter. 

This pull request fix the "problem"

### CODE REVIEW

**Comment by michelmegens** [comment]
Hey! First of all: thanks for your contribution!

If you want to, you can also update the version [here](https://github.com/cmdotcom/text-sdk-java/blob/faa2d5c5b242f4c5db374ae296289f935c6d8c90/pom.xml#L8) and the [changelog](https://github.com/cmdotcom/text-sdk-java/blob/master/CHANGELOG.md). Then we can release a new package immediately after merging your PR.

**Comment by algra4** [comment]
Hello @michelmegens 

Which version should I use, 2.1.0 or 2.0.2 ?

**Comment by michelmegens** [comment]
I'd go with 2.1.0, as it's technically a new feature of the/this SDK.

**Comment by algra4** [comment]
@michelmegens, I've changed the version

File: src/main/java/com/cm/text/models/Message.java [hunk 1]
```diff
@@ -140,6 +140,13 @@ public class Message {
     @SerializedName("validity")
     public String Validity;
 
+    /// <summary>
+    ///     Optional: If you set DCS to '0' or do not include the parameter, the messages uses standard GSM encoding. 
+    ///               To set your message up for unicode messaging set the value of dcs to 8 
+    ///     https://developers.cm.com/messaging/docs/sms#custom-dcs
+    /// </summary>
+    @SerializedName("dcs")
+    public String Dcs = "0";
```
Comment (michelmegens) [inline src/main/java/com/cm/text/models/Message.java]: Only just noticed, the DCS is a numerical value, not a string.

Comment (algra4) [reply src/main/java/com/cm/text/models/Message.java]: Done

**Review by michelmegens — APPROVED** [review]

**Pull request merged**